### ISSUE TITLE
Borg Removed from Homebrew?

### ISSUE THREAD

**Issue by Steve-MP** [borgbackup/borg#5335]
Hi,

The [installation instructions](https://borgbackup.readthedocs.io/en/stable/installation.html#distribution-package) recommend using ```brew cask install borgbackup ``` on Mac OSX

However, the borgbackup cask appears to be missing from the Homebrew repository, meaning that the installation instructions fail with the message
```
Error: Cask 'borgbackup' is unavailable: No Cask with this name exists.
```

See https://formulae.brew.sh/cask/borgbackup

**Comment by ThomasWaldmann** [comment]
Try brew install borgbackup (no "cask").

If the docs are not yet fixed in master / 1.1-maint branch, a PR for
that would be good.

> The installation instructions
> <https://borgbackup.readthedocs.io/en/stable/installation.html#distribution-package>
> recommend using |brew cask install borgbackup | on Mac OSX
> 
> However, the borgbackup cask appears to be missing from the Homebrew
> repository, meaning that the installation instructions fail with the message
> 
> |Error: Cask 'borgbackup' is unavailable: No Cask with this name exists. |
> 
> See https://formulae.brew.sh/cask/borgbackup
-- 

GPG ID: 9F88FB52FAF7B393
GPG FP: 6D5B EF9A DD20 7580 5747 B70F 9F88 FB52 FAF7 B393

**Comment by ThomasWaldmann** [comment]
https://borgbackup.readthedocs.io/en/1.1-maint/installation.html

The instructions were updated after 1.1.13 to be more generic (but requiring people to find out how to use their pkg manager and how the borg package is named), so the "brew cask borgbackup" is missing now.
